src/compass/utils/lut.py

```diff
@@ -1,4 +1,12 @@
-def compute_geocoding_correction_luts(burst, rg_step=200, az_step=0.25):
+'''
+Placeholder for model-based correction LUT
+'''
+import os
+
+import isce3
+
+def compute_geocoding_correction_luts(burst, rg_step=200, az_step=0.25,
+                                      dem_path=None, scratch_path=None):
```

Comment:
```suggestion
                                      dem_path, scratch_path=None):
```
Let's remove default value if this is a required parameter.

Reply:
I agree with this; and move the required `dem_path` arg location to the 2nd place.